internal-gateway/internal-gateway.nginx.conf

```diff
@@ -114,22 +87,20 @@ server {
         proxy_http_version 1.1;
         proxy_set_header Upgrade $http_upgrade;
         proxy_set_header Connection $connection_upgrade;
-
-        # we do not verify namepsaced routers
-        # include /ssl-config/ssl-config-proxy.conf;
```

Comment:
I'm inclined to leave this as a note to ourselves that we don't actually check the certs of the service